distribution/lib/Standard/Base/0.0.0-dev/src/Data/Any.enso

```diff
@@ -412,3 +413,14 @@ type Any
              2.to_default_visualization_data
     to_default_visualization_data : Text
     to_default_visualization_data self = self.to_json.to_text
+
+    ## UNSTABLE
+       ADVANCED
+
+       Returns the data requested to render a lazy view of the default visualisation.
+    to_lazy_visualization_data : Text
```

Comment:
I'm not sure if we should explicitly tell here that it is for "lazy" visualization. All of our visualizations should be lazy, so maybe it should be just named "to_visualization_data"?